rocAL/rocAL_pybind/rocal_pybind.cpp

```diff
@@ -28,6 +28,7 @@ THE SOFTWARE.
 #include <pybind11/eval.h>
 #include "rocal_api_types.h"
 #include "rocal_api.h"
+#include "tensor.h"
```

Comment:
we need to have another header file named "rocal_tensor.h" which exposes tensor attributes which is required for python

Reply:
Checked with Fiona, she said they have made these changes as a part of the CmakeLists changes